### ISSUE TITLE
Subtitle position in Stremio

### ISSUE THREAD

**Issue by Smith-Williams** [Stremio/stremio-features#962]
Enhancement Request: Subtitle Placement and Customization for PC Desktop Version

Dear Developers, I would like to suggest an enhancement to our current subtitle settings, specifically for the PC desktop version. It would be highly beneficial to provide an option for users to set subtitles at the top of the screen. This feature could greatly improve the viewing experience for many users, especially in cases where bottom-placed subtitles obstruct crucial visual content.

Moreover, offering a broader range of subtitle customization options would further enhance accessibility and user satisfaction. Features such as adjustable subtitle size, font choice, color options, and background shading can cater to diverse user preferences and needs.

I believe that implementing these enhancements will not only improve the overall user experience but also demonstrate our commitment to accessibility and user-centric design. I hope you will consider these suggestions for future updates.

Thank you for your attention to this matter.

**Comment by JumpIn-Git** [comment]
yes i would love this, we could also do it like youtube, where you can just drag the subtitles where you want it

**Comment by Smith-Williams** [comment]
> yes i would love this, we could also do it like youtube, where you can just drag the subtitles where you want it

Thanks! I try to reach with developers but no serious response ! 

I know that this feature it doesn't require that much work. It's simple and easy, but for some reason, they wouldn't do it and I don't know why

**Comment by Practicalbutterfly5** [comment]
Any update on this? I cannot change subtitle location on Mac as of today.
